cmd/githubwebhookserver/main.go

```diff
@@ -142,13 +142,19 @@ func main() {
 		}
 	})
 
+	// In order to support runner groups with custom visibility (selected repositories), we need to perform some GitHub API calls.
+	// Let the user define if they want to opt-in supporting this option by providing the proper GitHub authentication parameters
+	// Otherwise runner groups with custom visibility won't be supported and it will be assume all runner groups to be usable
+	// in all repositories to save API calls
```

Comment:
Thanks! This greatly helped me understand how it works. Here's my suggestion to improve it even further.

```suggestion
	// Without an opt-in, runner groups with custom visibility won't be supported to save API calls.
	// That is, all runner groups managed by ARC are assumed to be visible to any repositories,
	// which is wrong when you have one or more non-default runner groups in your organization or enterprise.
```